pkg/falcon_container_deployer/job_pullsecret.go

```diff
@@ -34,7 +34,7 @@ func (d *FalconContainerDeployer) GetJobSecret() (*corev1.Secret, error) {
 }
 
 func (d *FalconContainerDeployer) CreateJobSecret() error {
-	pulltoken, err := pulltoken.CrowdStrike(d.falconApiConfig())
+	pulltoken, err := pulltoken.CrowdStrike(d.Ctx, d.falconApiConfig())
```

Comment:
Are you seeing any error on missing context?

I think the context is already set-up in https://github.com/redhatrises/falcon-operator/blob/41cd1e0b22c9005e13d61435bcfa17df6154b7d5/pkg/falcon_container_deployer/image_push.go#L146